File: solutions-blind/python/003-luhn/solution_test.py
from solution import luhn_valid

assert luhn_valid("79927398713") is True
assert luhn_valid("79927398714") is False
assert luhn_valid("4532015112830366") is True
assert luhn_valid("7992-7398-713") is True
assert luhn_valid("") is False
assert luhn_valid("12a") is False
assert luhn_valid("0") is True
print("ok")

File: solutions-blind/javascript/003-luhn/solution.test.mjs
import test from "node:test";
import assert from "node:assert/strict";
import { luhn_valid } from "./solution.mjs";

test("luhn_valid", () => {
  assert.equal(luhn_valid("79927398713"), true);
  assert.equal(luhn_valid("79927398714"), false);
  assert.equal(luhn_valid("4532015112830366"), true);
  assert.equal(luhn_valid("7992-7398-713"), true);
  assert.equal(luhn_valid(""), false);
  assert.equal(luhn_valid("12a"), false);
  assert.equal(luhn_valid("0"), true);
});
